.github/workflows/enforceTestCoverage.yml

```diff
@@ -7,6 +7,9 @@ name: Enforce Min Test Coverage
 on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
+    paths:
+      - 'src/**.sol'
+      - 'test/**.sol'
```

Comment:
why do you add the folder here? you have other .sol file that are outside of `src/` and `test/` that you don't want to match this trigger?

Reply:
we always place .sol files in the `src/` and `test/` folders as this is the standard structure required by Foundry (development and testing environment for Solidity that we use). While `.sol` files can also exist in the `script/` folder those are intended for deployment or scripts etc and we do not need to run tests for them. That's why I put specifically only `src/` and `test/`.

Reply:
Didnt we say that we do not want to use the `paths` filter cause then there might be cases when the git action doesnt run (but since it is required this would cause problems)?

Reply:
I think using paths is fine. It should work as expected, and if not, we will adjust.

Reply:
Was discussed that we would use paths filter and remove "required" protection. IMO it won't work without removing "required" protection

Reply:
if we remove "required" then people can merge their PR even if this check fails which makes it useless

Reply:
I initially thought that marking a action as "required" meant that it must always run, but the final result could still block merging. That’s why I assumed there’s this "Expected - Waiting for status to be reported" status... makes sense